pkg/provisioner/ironic/ironic.go

```diff
@@ -433,6 +433,11 @@ func (p *ironicProvisioner) ValidateManagementAccess(data provisioner.Management
 	driverInfo["deploy_kernel"] = deployKernelURL
 	driverInfo["deploy_ramdisk"] = deployRamdiskURL
 
+	if data.CurrentImage != nil && data.CurrentImage.DiskFormat != nil && *data.CurrentImage.DiskFormat == "live-iso" {
```

Comment:
nit: There is `data.CurrentImage.IsLiveISO`, but only in 4.9.

Reply:
Yeah unfortunately we'll have to deal with those differences in the backport, my aim here is just to enable @mcornea to prove if this solves the specific problem encountered in his testing

Reply:
Ah I see actually that didn't yet merge, it's part of https://github.com/metal3-io/baremetal-operator/pull/909 - maybe @zaneb could take a look and potentially merge so I can base the master PR on that?